registrar-cli/cmd/create_account.go

```diff
@@ -0,0 +1,96 @@
+// Package cmd for parsing command line arguments
+package cmd
+
+import (
+	"crypto/ed25519"
+	"crypto/rand"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+	"github.com/threefoldtech/tfgrid4-sdk-go/node-registrar/client"
+)
+
+// createAccountCmd represents the cancel command
+var createAccountCmd = &cobra.Command{
+	Use:   "account",
+	Short: "create new account in node registrar",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		seed, err := cmd.Flags().GetString("seed")
+		if err != nil {
+			return err
+		}
+
+		network, err := cmd.Flags().GetString("network")
+		if err != nil {
+			return err
+		}
+
+		relays, err := cmd.Flags().GetStringArray("relays")
+		if err != nil {
+			return err
+		}
+
+		rmbEncKey, err := cmd.Flags().GetString("rmb-enc-key")
+		if err != nil {
+			return err
+		}
+
+		u, ok := urls[network]
+		if !ok {
+			return fmt.Errorf("invalid network %s", network)
+		}
+
```

Comment:
the function logic should be in pkg or internal and invoked here against the sanitized params